sample/Kmd.Logic.FileSecurity.Client.DocumentPrivilegesSample/Program.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Aspose.Pdf;
+using Aspose.Pdf.Facades;
+using Kmd.Logic.FileSecurity.Client.Models;
+using Kmd.Logic.FileSecurity.Client.ServiceMessages;
+using Kmd.Logic.Identity.Authorization;
+using Microsoft.Extensions.Configuration;
+using Serilog;
+
+namespace Kmd.Logic.FileSecurity.Client.DocumentPrivilegesSample
+{
+    public class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            InitLogger();
+
+            try
+            {
+                var config = new ConfigurationBuilder()
+                    .SetBasePath(AppContext.BaseDirectory)
+                    .AddJsonFile("appsettings.json", optional: false)
+                    .AddEnvironmentVariables()
+                    .AddCommandLine(args)
+                    .Build()
+                    .Get<AppConfiguration>();
+
+                await Run(config).ConfigureAwait(false);
+            }
+            catch (Exception ex)
+            {
+                Log.Fatal(ex, "Caught a fatal unhandled exception");
+            }
+            finally
+            {
+                Log.CloseAndFlush();
+            }
+        }
+
+        private static async Task Run(AppConfiguration configuration)
+        {
+            var validator = new ConfigurationValidator(configuration);
+            if (!validator.Validate())
+            {
+                return;
+            }
+
+            using var httpClient = new HttpClient();
+            using var tokenProviderFactory = new LogicTokenProviderFactory(configuration.TokenProvider);
+            var fileSecurityClient = new FileSecurityClient(httpClient, tokenProviderFactory, configuration.FileSecurityOptions);
+
+            // Get Sign Configuration
+            Log.Information("Getting signconfiguration...");
+            var signConfigurationResult = await fileSecurityClient.GetPdfSignConfiguration(configuration.SignConfigurationDetails.SignConfigurationId).ConfigureAwait(false);
+            if (signConfigurationResult == null)
+            {
+                Log.Error("Couldn't get signconfiguration");
+                return;
+            }
+
+            Console.WriteLine(
+                "Configuration recieved successfully. SignConfiguration ID: {0} \nSignConfiguration Name: {1}\nSubscription ID : {2}",
+                signConfigurationResult.Id,
+                signConfigurationResult.Name,
+                signConfigurationResult.SubscriptionId);
+
+            // Generate document
+            Log.Information("Generate document using privileges...");
+            string pdfSample = configuration.SignConfigurationDetails.PdfEmptySampleLocation;
+            using (Document document = new Document(pdfSample))
```

Comment:
Can we use the new `using` syntax here? e.g. `using (Document document = new Document(pdfSample));`

Reply:
Done